### PR TITLE
[wip] Raise priority of single-AZ kube workers

### PR DESCRIPTION
**Pull request by jonasschneider**
Sorry for the PEP8 spam!

### CODE REVIEW

**Comment by embreinhardt** [comment]
cc @vicki-c

**Comment by embreinhardt** [comment]
hooray for pep8! 🎉 

in the future, splitting no-op linting changes into a separate commit will make it easier to quickly identify the actual changes

**Comment by jonasschneider** [comment]
Sure. I just have this thing where everything is magically autoformatted whenever I save 😅  (don't hit me @vicki-c )

File: autoscaler/cluster.py [hunk 1]
```diff
@@ -337,8 +350,12 @@ def sort_key(group):
                 region = self.regions.index(group.region)
             except ValueError:
                 pass
-            priority = self._GROUP_PRIORITIES.get(group.selectors.get('aws/type'), self._GROUP_DEFAULT_PRIORITY)
-            return (region, not group.is_spot, priority, group.name)
+            # Some ASGs are pinned to be in a single AZ. Sort them in front of
+            # multi-ASG groups that won't have this tag set.
+            pinned_to_az = group.selectors.get('aws/az', 'z')
```
Comment (embreinhardt) [inline autoscaler/cluster.py]: @vicki-c for easy review, the relevant changes are on this line, and L358

**Comment by embreinhardt** [comment]
this looks good to me 👍

**Pull request merged**